manifests/base.pp

```diff
@@ -18,6 +18,9 @@
         }
       }
       'RedHat': {
+        if $facts['os']['release']['major'] == 8 {
+          fail('Upstream Repo does not currently work with EL8!')
```

Comment:
My concern here is that once it's fixed, we need to do another release and the user has no way around this. There is already a repo so clearly there is the intention from upstream. I'm leaning to a note in the README + changelog warning the user that the repo is currently broken. Sort a Known Issues thing.